app/dashboards/artist_dashboard.rb

```diff
@@ -69,7 +69,7 @@ class ArtistDashboard < Administrate::BaseDashboard
   # Overwrite this method to customize how artists are displayed
   # across all pages of the admin dashboard.
   #
-  # def display_resource(artist)
-  #   "Artist ##{artist.id}"
-  # end
+  def display_resource(artist)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)